doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -98,8 +98,8 @@ Images are stored using the OpenStack Glance service discussed below in Section
 The [OpenStack Storage Table](https://docs.openstack.org/arch-design/design-storage/design-storage-concepts.html#table-openstack-storage) explains the differences between the storage types and typical use cases. The [OpenStack compatible storage backend drivers](https://docs.openstack.org/cinder/latest/reference/support-matrix.html) table lists the capabilities that each of these drivers support.
 
 <a name="3.2.4"></a>
-### 3.2.4. Virtual Networking – Neutron standalone 
-Content to be developed
+### 3.2.4. Virtual Networking Neutron standalone 
+Neutron is an OpenStack project that provides "network connectivity as a service" between interface devices(e.g., vNICs) managed by other OpenStack services (e.g., nova). Neutron allows users to create networks, subnets, ports, routers etc and handles traffic isolation between different subnets - within as well as across project(s). Multiple network segments are supported by Neutron via ML2 plugins to simultaneously utilize varierty of layer 2 networking technologies like VLAN, VxLAN, GRE etc. Neutron also allows to create routers to connect layer 2 networks via "neutron-l3-agent". In addition, floating IP support is also provided that allows a project VM to be accessed using a public IP.
```

Comment:
@msidana Great start.
"... handles traffic isolation between different subnets .." -- I don't think neutron directly "handles traffic isolation"; mechanisms for traffic isolation are VLANs and tunneling.

Should some of the limitations and workarounds be mentioned?

Reply:
Neutron doesn't handles directly but facilitates such isolation by using different type drivers/mechanism drivers like VLANs, VxLANs, GRE (Generic Routing Encapsulation) tunnels etc. For neutron API consumer, this is abstracted and provided by neutron.

Reply:
@msidana Can you incorporate into the text? Thanks

Reply:
Done.